CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Master (unreleased)
 
+* Drop support for EOL'd rubies (under 2.3) by [@deivid-rodriguez][]
```

Comment:
Minor comment. I've seen us and other libraries start with the Ruby version as its easier to spot and stands out. Since we ran Travis builds for Ruby 2.2.x as the lowest, we can just specify that explicit version to keep it simple.

While writing this, I first checked out activeadmin's changelog to see what our original messaging is like since I preferred it and saw that you wrote all the recent ones. 😂 Consider:

> Ruby 2.2 support has been dropped.  by [@deivid-rodriguez][]

Reply:
The thing is... We're actually dropping support for all rubies under 2.3, not only 2.2. Only 2.2 was tested in TravisCI but the gem is currently installable on lower rubies and it might actually work. Up until now there was not a lower bound requirement in the gemspec file.

Reply:
Makes sense, thanks! I still think it’s fine since it’s what we used in Travis but that’s a personal preference. What you have is good and I understand now why it’s written that way since with no lowerbound we didn’t drop previously.

Reply:
I sometimes have done this distinction by saying  "Drop official support" (means I stop testing against that version but still allow installation and usage on older versions "at your own risk") vs just "Drop support", that also means disallowing installation of the gem on older rubies. It's normally simpler to do the latter and drop everything altogether.